spec/reek/code_comment_spec.rb

```diff
@@ -81,5 +81,15 @@
       config = described_class.new('# :reek: Duplication').config
       expect(config).not_to include('Duplication')
     end
+
+    it 'removes the configuration options from the comment' do
+      subject = described_class.new('
+        # Actual
+        # :reek:Duplication: { enabled: false }
+        # :reek:NestedIterators: { enabled: true }
+        # comment
+      ')
+      expect(subject.send(:sanitized_comment)).to eq('Actual comment')
```

Comment:
We don't!
However it makes debugging a lot easier and it's way more convenient to play around with stuff like this in the console.
Let's call it a productivity enhancement and I believe that we can't add enough of those ;)